client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -129,6 +141,10 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 			return (
 				<div>
 					<QuerySiteSettings siteId={ this.props.siteId } />
+					{
+						this.props.isJetpack &&
+						<QueryJetpackSettings siteId={ this.props.siteId } />
```

Comment:
Do you think we should add a check for `jetpackSettingsUISupported` here, or better inside the query component?